submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/itest/ITestListener.kt

```diff
@@ -98,6 +99,13 @@ class ITestListener : TestExecutionListener {
         System.setProperty("app.persistence.enableFire", "${System.getProperty("enableFire").toBoolean()}")
     }
 
+    private fun doiSetup() {
+        System.setProperty("app.doi.endpoint", "https://test.crossref.org/servlet/deposit")
```

Comment:
I was thinking we agree to use test doi in beta and use mock response in itest

Reply:
After reading the crossref documentation,[ their test system is actually a mock ](https://www.crossref.org/documentation/register-maintain-records/direct-deposit-xml/testing-your-xml/)that doesn't register anything so I guess it's ok to use it. We can discuss it in the meeting anyway.

Reply:
Yes, to avoid external dependencies. I just added a new wire mock for it